Dockerfile

```diff
@@ -10,7 +10,9 @@ RUN npm install
 
 ## If you are building your code for production
 ## RUN npm ci --only=production
-#
+
+ENV NODE_ENV=production
```

Comment:
Can we add NODE_ENV to .env file on the project?
Anybody can use Docker and it is not good to limit it to prod.

Reply:
When set the NODE_ENV to "production" in the .env file, it only affects how the application code behaves based on the specified environment. It doesn't change the installation process of packages when you run npm install.